march_hardware/include/march_hardware/Joint.h

```diff
@@ -12,14 +12,15 @@ class Joint
 {
 private:
   std::string name;
+  bool actuate;
```

Comment:
Suggestion: change variable name to actuateEnabled, which describes it better imo

Reply:
Yeah I thought about this. Something like `canActuate` or `allowActuation` or `actuatePossible` are also options. I went with actuate because it looks nice in the yaml `actuate: true` but it can be changed for sure.

Reply:
@TimBuckers thoughts?

Reply:
allowActuation or actuateEnable is a good solution I think

Reply:
@Ishadijcks I believe in the variable name allowActuation, should we do that?

Reply:
Sure